include/soci/blob.h

```diff
@@ -27,32 +27,45 @@ class blob_backend;
 class SOCI_DECL blob
 {
 public:
+    // Creates an invalid blob object
+    explicit blob() = default;
```

Comment:
I'm not sure if this is intentional, AFAIK this does have a very minor effect (it prohibits writing `blob b = {};`, which otherwise would be the same as `blob b;`), but is pretty unusual, so if it wasn't, I'd just change it to

```suggestion
    blob() = default;
```

Reply:
Good point